lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -19,30 +18,34 @@ export default function LMSGrader({
   assignmentName,
   courseName,
   students,
-  onChangeSelectedUser,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  /**
+   * Makes an RPC call to the sidebar to change to the focused user.
+   *
+   * @param {User} user - The user to focus on in the sidebar
+   */
+  const changeFocusedUser = async user => {
+    const sidebar = await getSidebarWindow();
+    // Calls the client sidebar to fire the `changeFocusModeUser` action
+    // to change the focused user.
+    rpcCall(sidebar.frame, sidebar.origin, 'changeFocusModeUser', [
+      {
+        username: user.userid, // change `username` key to `userid` once the client is ready
+        displayName: user.displayName,
+      },
+    ]);
+  };
+
   useEffect(() => {
     if (students[currentStudentIndex]) {
-      // set focused user
-      updateClientConfig({
-        focus: {
-          user: {
-            username: students[currentStudentIndex].userid,
-            displayName: students[currentStudentIndex].displayName,
-          },
-        },
-      });
-      // let the parent component know the index changed
-      onChangeSelectedUser(students[currentStudentIndex].userid);
+      changeFocusedUser(students[currentStudentIndex]);
     } else {
-      // clear focused user
-      removeClientConfig(['focus']);
-      onChangeSelectedUser('0'); // any non-real userid will work
+      changeFocusedUser({}); // any non-real userid will clear out a previously focused user
```

Comment:
Note, no need to keep track of a cancel var since we're not setting state as like was done here
https://github.com/hypothesis/lms/blob/master/lms/static/scripts/frontend_apps/components/SubmitGradeForm.js#L25